packages/cfd/src/Containers/migration-banner/migration-banner.tsx

```diff
@@ -58,7 +67,7 @@ const MigrationBanner = observer(() => {
                                     ),
                                     platform: getCFDPlatformNames(CFD_PLATFORMS.MT5),
                                 }}
-                                components={[<br key={0} />, <strong key={1} />]}
+                                components={[!is_desktop_trade_modal && <br key={0} />, <strong key={1} />]}
```

Comment:
just I'm curious:
if the 'is_desktop_trade_modal' is true, then inside components array we will have [ false, <strong key={1} />] right?
is iit working correctly?
as I see elements inside components can be `readonly React.ReactElement[] | { readonly [tagName: string]: React.ReactElement }`